### PR TITLE
Update rio badge

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

Updates the rio logo badge

### CODE REVIEW

**Comment by TBonnin** [comment]
my svg parsing skill is quite rusty but 👍  💃

File: static/public/images/badges/rio-2016.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...2.216-22.423 5.476v-5.867c6.78-3.65 13.818-5.996 24.118-5.996 17.47 0 26.987 9.125 26.987 25.03 0 16.036-10.95 27.77-26.205 42.632l-17.86 17.47v.26h46.28v6.258h-55.537v-5.477zM457.69 308.705v-4.172c0-30.246 14.603-44.717 35.2-44.717 20.47 0 34.68 14.21 34.68 44.326v4.172c0 30.115-13.69 44.456-34.81 44.456-20.99 0-35.07-14.6-35.07-44.065zm62.318.522v-5.215c0-24.12-8.604-38.068-27.117-38.068-18.51 0-27.376 13.56-27.376 37.547v5.347c0 23.988 8.865 37.807 27.247 37.807 18.513 0 27.248-12.908 27.248-37.417zM548.688 268.682l-17.34 3.26v-4.434l21.773-7.17h2.868v91.13h-7.3V268.68zM571.37 312.355v-3.91c0-36.113 17.73-48.63 39.374-48.63 7.56 0 13.818 1.304 17.73 3.39v5.867c-3.912-1.564-10.04-2.868-16.558-2.868-17.99 0-31.81 7.952-32.724 39.11 5.477-4.432 14.602-8.473 26.205-8.473 16.297 0 28.55 8.214 28.55 26.466 0 17.73-11.08 29.464-29.593 29.464-21.77 0-32.983-16.687-32.983-40.415zm54.888 11.604c0-16.037-9.518-20.99-22.814-20.99-10.69 0-18.514 3.65-24.25 7.952v1.564c0 19.555 7.04 34.156 25.03 34.156 13.69 0 22.034-7.69 22.034-22.683z"/><path fill="#FBE400" d="M180.62 250.535L3.595 127.235 180.62 3.934l177.023 123.3"/><g><circle fill="#21449B" cx="179.843" cy="125.427" r="54.64"/></g></svg>
```
Comment (piuccio) [inline static/public/images/badges/rio-2016.svg]: I think you need `xmlns="http://www.w3.org/2000/svg"`

Comment (piuccio) [inline static/public/images/badges/rio-2016.svg]: well, unless you know you're only ever going to inline it

Comment (gtrufitt) [inline static/public/images/badges/rio-2016.svg]: Yeah, you're right, it's not inlined

File: static/public/images/badges/rio-2016.svg [hunk 2]
```diff
@@ -1 +1 @@
...
```
Comment (NataliaLKB) [inline static/public/images/badges/rio-2016.svg]: You need to remove the xmins stuff

Comment (NataliaLKB) [inline static/public/images/badges/rio-2016.svg]: Nevermind!

**Comment by paperboyo** [comment]
All SVGs could be smaller by having just one decimal place (this could be 30% smaller). https://jakearchibald.github.io/svgomg/ your friend. But it's only 2,5KB + it's Friday. Have a nice weekend!

**Comment by gtrufitt** [comment]
@paperboyo I did use svgomg and shaved off 25% but that's a good tip! Will trim it down Monday 👍

**Pull request merged**